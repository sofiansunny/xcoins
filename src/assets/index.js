const USD_ICON = require('./icons/US.png');
const GBP_ICON = require('./icons/GB.png');
const EURO_ICON = require('./icons/EU.png');
const TRANSFER_ARROW = require('./icons/transfer_arrow-icon.png');

export {USD_ICON, GBP_ICON, EURO_ICON, TRANSFER_ARROW};
