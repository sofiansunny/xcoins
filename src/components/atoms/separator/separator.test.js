import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Separator from '.';
import {EURO_ICON} from '../../../assets';

describe('SomeComponent component', () => {
  it('Shallow rendering', () => {
    const props = {
      style: {},
    };
    const wrapper = shallow(<Separator {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
