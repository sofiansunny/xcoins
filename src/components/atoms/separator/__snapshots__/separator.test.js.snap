// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomeComponent component Shallow rendering 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#f2f2f2",
        "height": 1.9617647058823529,
        "width": "100%",
      },
      Object {},
    ]
  }
/>
`;
